src/GafferImage/OpenColorIOTransform.cpp

```diff
@@ -112,6 +130,36 @@ OpenColorIO::ConstContextRcPtr OpenColorIOTransform::getLocalContext(OpenColorIO
 {
 
 	OpenColorIO::ConstContextRcPtr context = config->getCurrentContext();
+	const CompoundDataPlug *p = contextPlug();
+	if(!p)
+	{
+		return context;
+	}
+
+	if( !p->children().size() )
+	{
+		return context;
+	}
+
+	OpenColorIO::ContextRcPtr mutableContext;
+	std::string name;
+	std::string value;
+
+	for( CompoundDataPlug::MemberPlugIterator it( p ); !it.done(); ++it )
+	{
+		IECore::DataPtr d = p->memberDataAndName( it->get(), name );
+		if( d )
+		{
+			value = runTimeCast<StringData>(d)->readable();
```

Comment:
If the runTimeCast fails, let's throw an exception saying that only strings are supported. That way if the user does add something they shouldn't (maybe via scripting, or because we don't get the UI done in time for a merge), they'll get a helpful error in the graph.